src/components/AppSidebar/AppSidebar.vue

```diff
@@ -82,13 +82,37 @@
 		}
 	</script>
 	```
+	```
+
+	### Blank sidebar for e.g. empty content
+	```vue
+	<template>
+		<AppSidebar
+			:title="title"
+			blank="true">
+			<EmptyContent icon="icon-search">
+				Content not found.
+			</EmptyContent>
+		</AppSidebar>
+	</template>
+	<script>
+		export default {
+			data() {
+				return {
+					title: 'cat-picture.jpg'
+				}
+			}
+		}
```

Comment:
We could just put this as an inline string for the example :)

Reply:
Fixed.